components/mpas-ocean/src/shared/mpas_ocn_time_average_coupled.F

```diff
@@ -116,6 +118,33 @@ subroutine ocn_time_average_coupled_init(forcingPool)!{{{
            !$omp end parallel
         end if
 
+        ! Set up polar fields if necessary
+        if(trim(config_land_ice_flux_mode)=='standalone' .or. 'data') then
```

Comment:
@darincomeau, same issue here as below.  This may explain why you still were having trouble with `data` mode.

Reply:
Yeah this is fixed locally, I just hadn't pushed it yet since I was still getting 0s.